week1solution.py/solution6.py

```diff
@@ -0,0 +1,5 @@
+s=input("enter a string:")
+
+t=int(input("enter start position:"))
+e=int(input("enter end position:"))
+print(f"the part of the string '{s}' is '{s[t-2:e:]}'")

```

Comment:
explain this line .

Reply:
here i use slice function for sliceing string